data/campaigns/Son_Of_The_Black_Eye/scenarios/05_To_the_Harbor_of_Tirigaz.cfg

```diff
@@ -123,13 +123,13 @@
 
     [story]
         [part]
-            story= _ "Kapou’e and the shamans rested and discussed the matter for three days. It was finally decided that it might be necessary to call up the Great Horde to deal with this massive human incursion. However, in order to do that the entire Great Council would have to gather."
+            story= _ "Kapou’e and the shamans rested and discussed the matter for three days. It was finally decided that it might be necessary to call up the Great Horde to deal with this massive human incursion. However, in order to do that, the entire Great Council would have to gather."
             {SOTBE_BIGMAP}
             {JOURNEY_04_OLD}
         [/part]
 
         [part]
-            story= _ "Since it was now obvious that the humans were going to try their best to prevent that by capturing or killing the shamans, Kapou’e was requested to escort the shamans to the harbor of Tirigaz where the rest of the Council was residing."
+            story= _ "Since it was now obvious that the humans were going to try their best to prevent that by capturing or killing the shamans, Kapou’e was requested to escort the shamans to the Harbor of Tirigaz where the rest of the Council was residing."
```

Comment:
I made harbour lower-case quite recently in #6067 because Tirigaz is described as a port city rather than just a harbour (see 95a5bba9889ebe82ee412d096342879684280026).

To avoid confusion, perhaps it can be reworded to 'escort the shamans to the harbour **in** Tirigaz...'

Reply:
I agree with harbor of Tirigaz. You are absolutely right, there is even a scenario in Dead Water called "Tirigaz". It just entered my head to capitalize Harbor of Tirigaz because of the title case of the scenrio name "To the Harbor of Tirigaz".

Reply:
Yeah, not unreasonable. Hence my suggestion to rephrase it.